build/rust/config.gni

```diff
@@ -23,6 +23,9 @@ if (is_win) {
   if (current_cpu == "x64") {
     rustc_target = "x86_64-apple-darwin"
   }
+  else if (current_cpu == "arm64") {
```

Comment:
I would have placed the `else if` on the same line (25) as the preceding curly brace.

Reply:
done